crud/Generator.php

```diff
@@ -144,6 +150,7 @@ public function rules()
             parent::rules(),
             [
                 [['providerList'], 'filter', 'filter' => 'trim'],
+                [['fileFieldMatches'], 'filter', 'filter' => 'trim'],
```

Comment:
The previous line has not been modified in order to have a clean merge with [this other change](https://github.com/schmunk42/yii2-giiant/pull/103/files#diff-8a387a87a18bd6fb8ee76137e010b0d5L146)